draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -960,6 +986,9 @@ HTTPS, and receiving an HTTPSSVC record is intended to be similar to
 receiving that field value over HTTPS.  However, there are some
 differences in the intended client and server behavior.
 
+Unlike Alt-Svc, the ALPN value in a SVCB record is not
+guaranteed to match the ALPN value negotiated in the TLS handshake.
```

Comment:
I would prefer to correct Alt-Svc on this point.  But that requires more work.  Perhaps you could leave a marker here indicating a desire to fix that aspect of 7838.

Reply:
OK, I added a marker.